src/ucp/wireup/select.c

```diff
@@ -979,9 +1006,9 @@ ucp_wireup_add_am_lane(const ucp_wireup_select_params_t *select_params,
         criteria.local_iface_flags |= UCP_WORKER_UCT_UNSIG_EVENT_CAP_FLAGS;
     }
 
-    status = ucp_wireup_select_transport(select_params, &criteria, UINT64_MAX,
-                                         UINT64_MAX, UINT64_MAX, UINT64_MAX,
-                                         1, am_info);
+    status = ucp_wireup_select_transport(select_params, &criteria,
```

Comment:
why needed in this PR?

Reply:
Need to define tl_bitmap bound to specific device